packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/home/home.html

```diff
@@ -0,0 +1,144 @@
+<div>
+    <div class="row my-2">
+        <!-- AI Pipeline-->
+        <div class="col-lg-8 col-12 mb-2">
+            <div class="row mb-4">
+                <div class="col">
+                    <pci-project-ai-pipeline
+                        data-ai-items="$ctrl.aiItems"
+                        data-on-storage-click="$ctrl.trackAndGoToObjectStorage()"
+                        data-on-notebook-click="$ctrl.trackAndGoToAINotebook()"
+                        data-on-training-click="$ctrl.trackAndGoToAITraining()"
+                        data-on-deploy-click="$ctrl.trackAndGoToAIDeploy()"
+                    >
+                    </pci-project-ai-pipeline>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col mb-2">
+                    <!--Billing-->
+                    <div class="oui-box-block">
+                        <div class="oui-box-block__heading">
+                            <span
+                                data-translate="pci_ai_dashboard_home_billing_title"
+                            ></span>
+                        </div>
+                        <div class="oui-box-block__body">
+                            <div
+                                class="oui-box-block__body__content d-flex flex-column"
+                            >
+                                <strong
+                                    data-translate="pci_ai_dashboard_home_billing_next_billing"
+                                ></strong>
+                                <span
+                                    data-translate="pci_ai_dashboard_home_billing_next_billing_data"
+                                    data-translate-values="{ date: ($ctrl.nextBillingDate | date: 'mediumDate') }"
+                                ></span>
+                                <span
+                                    class="billing-price"
+                                    data-ng-bind="$ctrl.price | currency"
```

Comment:
**nitpick:** Could you please use `catalog-price`?

Reply:
Sure, done here: https://github.com/ovh/manager/pull/8172/commits/048b5d94917610f8c6ac0a2ab3c9501627b41dfa